examples/src/multiplex/server.rs

```diff
@@ -23,7 +23,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let addr = "[::1]:50051".parse().unwrap();
 
     let greeter = GreeterServer::new(MyGreeter::default());
-    let echo = EchoServer::new(MyEcho::default());
+    let echo = EchoServer::new(MyEcho);
```

Comment:
For some reason not all `default_constructed_unit_structs` targets are detected, resulting in partial changing and a lack of consistency throughout the code. If it seems difficult to resolve, I think we can allow this rule for the examples as the crate level setting.